packages/genesys-spark-components/src/components/stable/gux-table-toolbar/gux-table-toolbar.tsx

```diff
@@ -259,7 +259,11 @@ export class GuxTableToolbar {
 
   render(): JSX.Element {
     return (
-      <Host role="toolbar" aria-orientation="horizontal">
+      <Host
+        role="toolbar"
+        aria-orientation="horizontal"
+        gs-layout={this.displayedLayout}
```

Comment:
Out of the two solutions I would +1 for this as its a more encapsulated. Would there be any downside to exposing this `gs-layout` attribute though ?